runtime/onert/backend/train/BackendContext.cc

```diff
@@ -30,9 +30,9 @@ namespace train
 
 ITensorRegistry *BackendContext::genTensors() { return basic::genTensors(*this); }
 
-FunctionMap BackendContext::genKernels()
+backend::FunctionMap BackendContext::genKernels()
 {
-  FunctionMap ret;
+  backend::FunctionMap ret;
```

Comment:
There was a conflict with the namespace train in the core, so I added `backend::`.